src/OrbitBase/include/OrbitBase/Chunk.h

```diff
@@ -0,0 +1,47 @@
+// Copyright (c) 2022 The Orbit Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+#ifndef ORBIT_BASE_CHUNK_H_
+#define ORBIT_BASE_CHUNK_H_
+
+#include <absl/types/span.h>
+#include <stddef.h>
+
+namespace orbit_base {
+
+// Chunk an input vector into spans of specified size. Note that the last span can be smaller
+// than the specified "chunk_size" if the input vector size is not a multiple of "chunk_size".
+//
+// Example: Splitting data in chunks to be processed in parallel tasks:
+//
+// void ProcessObjectsInParallel(std::vector<Object>& objects) {
+//   TaskGroup task_group;
+//   for (absl::Span<Object> chunk : Chunk(objects, 1024)) {
```

Comment:
```suggestion
//   for (absl::Span<Object> chunk : CreateChunksOfSize(objects, 1024)) {
```

Reply:
Ah nice, thanks for catching this, I initially renamed "CreateSpansOfSize" -> "Chunk", but that was too minimalistic, it was confusing because we didn't know if the parameter was the size of the chunk or the number of chunks. Will change the comment.